sequence.go

```diff
@@ -125,3 +125,37 @@ func RandomProteinSequence(length int, seed int64) (string, error) {
 
         return string(randomSequence), nil
 }
+
+//RandomDNASequence returns random DNA sequences with specified length. 
+
+package main 
+
+import "fmt"
+import gonum as np
+func random_dna_sequence(length, gc_share=None, probas=None, seed=None) {
```

Comment:
exported function names in Go don't have underscores `RandomDna` will be a fine name here.

Also there's no default variable setting in Go. Nor is there a None object.